jobs/telemetry_batch_view.py

```diff
@@ -92,6 +110,9 @@ def update_metastore(location, hive_server):
 if environ.get("DO_RETRIEVE", "True") == "True":
     retrieve_jar()
 
+if environ.get("DO_EVENTS_TO_AMPLITUDE_SETUP") == "True":
+    events_to_amplitude_setup(environ.get("TBV_config_file_path"), environ.get("KEY_BUCKET"), environ.get("KEY_PATH"))
```

Comment:
I think to match the style of `retrieve_jar()` above, this should be a function with no arguments, and the values should be extracted from `environ` in the body of `events_to_amplitude_setup`